### PR TITLE
Include the response text in RequestError

### PR DESCRIPTION
**Pull request by kalikiana**
See: https://progress.opensuse.org/issues/120939

### CODE REVIEW

**Comment by perlpunk** [comment]
@AdamWill is there a command (e.g. black, ...) that one can use locally to fix the pylint errors?

**Comment by AdamWill** [comment]
Well, the only issue is one thing:

      tests/test_client.py:383: C0301: (line-too-long), : Line too long (148/100)

just split that line somehow and you're good. In this case running `black` would likely do it, yes. `black` doesn't always resolve pylint issues, though - they're complementary in some ways, which is why I run them both.

**Comment by perlpunk** [comment]
Hm, I tried to call `black` but it didn't do any changes:
```
% black src/ tests/  --exclude const.py  # the line from tox.ini
All done! ✨ 🍰 ✨
5 files left unchanged.
```
I personally wouldn't know what's the best practice here to split up that string, so I wonder if we can just tell black the maximum line length in some configuration file.

**Comment by kalikiana** [comment]
> I personally wouldn't know what's the best practice here to split up that string, so I wonder if we can just tell black the maximum line length in some configuration file.

I updated this line now using `--experimental-string-processing -l 90` but also opened #36 for other changes since there's a lot more places where lines are too long as far as black's concerned.

**Comment by mimi1vx** [comment]
hmm, 3.6 is more than year out of support and 3.11 isnt alpha anymore ( 3.12 is going to beta phase ...)

**Comment by perlpunk** [comment]
Regarding the line length: looking at the test again, the test string doesn't have to be that long, so it would also be more readable if you just used a short dummy string.

File: tests/test_client.py [hunk 1]
```diff
@@ -380,13 +380,18 @@ def test_get_jobs(self, fakerequest, fakeclones, simple_config):
 
     def test_client_errors(self):
         """Test creation of exceptions"""
-        err = oqe.RequestError("GET", "http://localhost", 404)
+        template_error = (
```
Comment (perlpunk) [inline tests/test_client.py:383]: I suggest to use a short dummy text here instead. Then no line splitting is needed

Comment (AdamWill) [reply tests/test_client.py:383]: yeah, agreed. I do always like "real" data in tests when possible, but if it's just an arbitrary string, no need to cause ourselves pain.

**Comment by AdamWill** [comment]
Not sure what's going on with the Python 3.6 thing. I tend just reflexively to keep it in the list for my projects because it's what RHEL 8 has for a system Python, but in this project that probably doesn't matter. Still, I'd like to know *why* it can't find "3.6" - the list it links to has plenty of 3.6 versions in it. It doesn't list a literal "3.6", but it doesn't list a literal "3.7" or "3.8" or "3.9" (etc.) either and those work. I'll look into it a bit and fix it one way or another.

**Comment by AdamWill** [comment]
OK, I guess this is fine. Thanks.

**Pull request merged**